### DISCUSSION TITLE
First hackathon

### DISCUSSION THREAD

**Discussion by HUSK1E** [github/hackathons#5] [General]
What was the first hackathon you organised?

What did you learn from it?

**Comment by mishmanners** [comment]
Ooooo, I like this one. My first hackathon I organised was back in 2015. I turned it around in less than a month. I learned, you need more time than that!

I also learned what works and what doesn't. I wrote about a lot of it on my blog. I'll find it and post it here later. Thanks for the great topic @HUSK1E

**Comment by upsilon1** [comment]
I recently participated in the 2022 hackathon that was organized by the company I work for. This was an internal event, the first one in the company. And it was really awesome! 🚀

We were competing for a prize (money) 🏆 And the duration of the event was larger than usual hackathons: more than a month.

Our boss wanted us to obtain new skills, try new roles, get a product mindset, and a better understanding of the path that startups go through when launching products. And he was right, the 4 teams we had all note that they’ve learned so much from the experience! 👨‍💻

If anyone’s interested in the event details (with photos and quotes from the teams regarding the challenges and obtained knowledge), I’m sharing this post 👉 [https://www.linkedin.com/pulse/upsilons-first-hackathon-retrospective-upsilon-it](https://www.linkedin.com/pulse/upsilons-first-hackathon-retrospective-upsilon-it)
